library/core/src/num/nonzero.rs

```diff
@@ -287,6 +287,204 @@ nonzero_integers_div! {
     NonZeroUsize(usize);
 }
 
+macro_rules! nonzero_integers_add {
+    ( $( $Ty: ident($Int: ty); )+ ) => {
+        $(
+            impl $Ty {
+                /// Add a positive integer to a non-zero value.
+                /// Return [`None`] on overflow.
+                ///
+                /// # Examples
+                ///
+                /// ```
+                /// # #![feature(nonzero_add)]
```

Comment:
We usually keep the feature flags visible in the examples.
```suggestion
                /// #![feature(nonzero_add)]
```

Reply:
Okay, what about the `use` lines? They seem present on top of the page, but not in every function doctest. I'm leaving them hidden for now.